cms/templates/js/paging-header.underscore

```diff
@@ -1,6 +1,6 @@
 <div class="meta-wrap">
     <div class="meta">
-        <%= messageHtml %>
+        <p><%= messageHtml %></p>
```

Comment:
For the safe template linter to be happy with this, you should use:

    <p><%= HtmlUtils.ensureHtml(messageHtml) %></p>